Validation/Configuration/python/globalValidation_cff.py

```diff
@@ -95,6 +97,8 @@
                                  + bTagPlotsMCbcl
                                  + L1Validator
                                  + bdHadronTrackValidationSeq
+                                 + OuterTrackerSourceV
```

Comment:
this looks misplaced. phase-2 specific non-empty modules/sequences have no place in common sequences

Reply:
Would adding it to "globalValidationTrackingOnly" be OK? Or should it only be in specific phase 2 sequences?